app/services/katello/content_unit_indexer.rb

```diff
@@ -55,6 +55,15 @@ def import_all(filtered_indexing = false)
       end
     end
 
+    def import_manifest_labels
```

Comment:
This method ended up being totally generic, so what do you think about somethink like:

```suggestion
    def reimport_by_type
```

Reply:
Good point..I landed at this version after trying some different ways. Didn't realize it became generic. :)